.azurepipelines/BaseTools-Build-For-Publication.yml

```diff
@@ -0,0 +1,146 @@
+name: BaseTools_Bin_$(release_version)_$(Rev:r)
+
+variables:
+  linux_build_output_path: '$(Build.Repository.LocalPath)/BaseTools/Source/C/bin'
+  linux_artifact_name: 'LinuxOutput'
+  windows_build_output_path: '$(Build.Repository.LocalPath)\BaseTools\Bin\Win32'
+  windows_artifact_name: 'WindowsOutput'
+  temp_publication_directory: 'PackageStaging'
+  package_artifact_name: 'NugetReleasePackage'
+
+jobs:
+  - job: linux_build
```

Comment:
there is a ton of duplication with windows job.  I think we could use matrix and parameters and handle this

Reply:
Agreed. Will open an issue and push to second version.